<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stoptime_dialog</class>
 <widget class="QDialog" name="stoptime_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="stoptime_label">
       <property name="text">
        <string>Enter / Quit or dial; Stops at</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="stoptime_dateTimeEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd    HH:mm:ss</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="duration_label">
       <property name="text">
        <string>dd  hh : mm : ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>days</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>hours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>minutes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>seconds</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="day_spin">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hour_spin">
       <property name="maximum">
        <number>23</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minute_spin">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="second_spin">
       <property name="prefix">
        <string notr="true"/>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>stoptime_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>stoptime_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
